content/en/docs/components/pipelines/v2/cli.md

```diff
@@ -4,119 +4,161 @@ description = "Interact with KFP via the CLI"
+### Interact with KFP resources<a id="cli_interact_with_kfp_resources"></a>
+
+The majority of the KFP CLI commands let you create, read, update, or delete KFP resources from the KFP backend. All of these commands use the following general syntax:
 
 ```shell
 kfp <resource_name> <action>
 ```
 
-`<resource_name>` can be one of the following:
+The `<resource_name>` argument can be one of the following:
 * `run`
 * `recurring-run`
 * `pipeline`
 * `experiment`
 
-All resources support the following actions:
+For all values of the `<resource_name>` argument, the `<action>` argument can be one of the following:
 * `create`
 * `list`
 * `get`
 * `delete`
 
-Some resources have resource-specific actions, including:
-* For `run`:
-  * `archive`
-  * `unarchive`
+Some resource names have additional resource-specific actions. The following table lists a few examples of resource-specific actions:
 
-* For `recurring-run`:
-  * `disable`
-  * `enable`
+| Resource name | Additional resource-specific actions
+|---------------|--------
+| `run` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `recurring-run` | <ul><li>`disable`</li><li>`enable`</li></ul>
+| `experiment` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `pipeline` |  | <ul><li>`create-version`</li><li>`list-versions`</li><li>`get-versions`</li><li>`delete-versions`</li></ul>
 
-* For `experiment`
-  * `archive`
-  * `unarchive`
+### Compile pipelines<a id="cli_compile_pipelines"></a>
 
-* For `pipeline`:
-  * `create-version`
-  * `list-versions`
-  * `get-versions`
-  * `delete-versions`
+You can use the `kfp dsl compile` command to compile pipelines or components defined in a Python file to IR YAML.
 
+* To compile a pipeline defined in a Python file, run the following command:
 
-## Compile pipelines
-The KFP supports compiling a pipeline defined in a Python file to YAML using the `kfp dsl compile` command:
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml
+  ```
+<!-- TODO: Validate or review these commands -->
+* To compile a pipeline from a Python file containing multiple pipeline definitions, use the `--function` argument:
 
-```shell
-kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml
-```
+  ```shell
+  kfp dsl compile --py [PATH_TO_INPUT_PYTHON] --output [PATH_TO_OUTPUT_YAML] --function my_pipeline
+  ```
 
-To compile a pipeline from a Python file containing multiple pipeline definitions or to compile a component, provide a `--function` argument:
+  For example:
 
-```shell
-kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --function my_component
-```
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml --function my_pipeline
+  ```
+
+* To compile a component from a Python file containing multiple pipeline or component definitions, use the `--function` argument:
```

Comment:
This bullet seems duplicative of the one above, with only a subtle difference. Can they be collapsed into one, since the user action for handling for multiple pipelines and components is the same?

Reply:
Made an edit to fix this.